apps/codeforafrica/src/components/Footer/Footer.js

```diff
@@ -40,7 +46,7 @@ const Footer = React.forwardRef(function Footer(props, ref) {
                 item
                 xs={12}
               >
-                <FooterDescription description={description} />
+                <FooterDescription description={description} logo={logo} />
```

Comment:
Convert `description` to `children` so that we can write this as:
```jsx
                <FooterDescription logo={logo}>
                  {description}
                </FooterDescription>
```